eng/scripts/Language-Settings.ps1

```diff
@@ -89,10 +89,16 @@ function Get-dotnet-PackageInfoFromPackageFile ($pkg, $workingDirectory)
   }
 }
 
+# Return list of nupkg artifacts
+function Get-dotnet-Package-Artifacts ($Location)
+{
+  return Get-ChildItem "$($Location)" -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+}
```

Comment:
We should have this return the one file and verify there is only one nupkg file in the location. The existing usages assume there should only be one so we should just enforce it here.